py/specs/in_string.toml

```diff
@@ -0,0 +1,17 @@
+[setup]
+statements = ["create table t_in_str (id int primary key, value string) with (number_of_replicas=0)"]
+
+     [[setup.data_files]]
+        target = "t_in_str"
+        source = "data/id_int_value_str.json"
+
+
+[[queries]]
+statement = "select * from t_in_str where value in (?,?,?,?,?,?,?,?,?,?)"
+args = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'] 
+concurrency = 1
+iterations = 100000
```

Comment:
as value is a string field you should use the amplab uservisits table (e.g. "sourceIP" or "destinationURL" ) and reduce the iterations. ( it's preferable to have longer query times than 100000 iterations)